pynucastro/networks/rate_collection.py

```diff
@@ -2484,12 +2542,20 @@ def get_node_color(_nuc):
         if rate_ydots is not None:
             pc = mpl.collections.PatchCollection(real_edges_lc, cmap=plt.cm.viridis)
             pc.set_array(real_weights)
-            fig.colorbar(pc, cax=rate_cb_ax, label="log10(rate)", orientation=orientation)
+            label = r"$\log_{10}(\mathrm{rate})$"
+            if use_net_rate:
+                if normalize_net_rate:
+                    label = r"$\log_{10}((\lambda_\mathrm{forward} - \lambda_\mathrm{reverse}) / [\frac{1}{2} (\lambda_\mathrm{forward} + \lambda_\mathrm{reverse}))]$"
```

Comment:
I think `()` and `[]` are mismatched. Should swap the last `]` and `)` before that.

Reply:
yep.  fixed